src/test/java/com/example/demo/services/MessageServicesTest.java

```diff
@@ -0,0 +1,54 @@
+package com.example.demo.services;
+
+import com.example.demo.model.Message;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MessageServicesTest {
+    MessageServices ms = new MessageServices();
+    private Message message =new Message("today22");;
+
+
```

Comment:
try to format text by default guidelines or ask Idea to do it for you at least)
ctrl + alt + L